File: 8KYU/Calculate_average.js
/*
function which calculates the average of the numbers in a given list.
Empty arrays should return 0.
*/

let findAvarage = (arr) =>{
    let sum = arr.reduce( (a,b) => a+b, 0);
    let avrg = (sum / arr.length) || 0;
    return avrg
}

File: C/8KYU/calculate_average.c


/**
 *  function which calculates the average of the numbers in a given list.
 */

double find_average(const double array[], unsigned length) {
  double sum = 0;
  for (unsigned i = 0; i < length; i++) {
    sum += array[i];
  }
  if (length > 0) {
    double average = sum / length;
    return average;
  } else {
    return 0;
  }
}
